NodeRequirer.sublime-settings

```diff
@@ -36,5 +36,7 @@
     },
     // Use 'single' or "double" quotes
     "quotes": "single",
+    // Use 'var' or 'let' variable declarations
```

Comment:
The other part of the code only targeted let and var, but I agree. I'll make this patch also add support for const everywhere.